psi4/driver/procrouting/proc.py

```diff
@@ -3008,7 +3008,7 @@ def run_cc_property(name, **kwargs):
     if n_one > 0:
         # call oe prop for GS density
         oe = core.OEProp(ccwfn)
-        oe.set_title("CC")
+        oe.set_title(name.upper())
```

Comment:
`CC DIPOLE` wfn var is set by OEProp, so just had to push the proper title through `prop.py`. I would have used the above-defined `this_name` variable, but EOM needs to keep the `-`. 

The `Process::environment` variables are a little more annoying to change. Shall we keep them as is to keep from breaking downstream, or insist on consistency btwn Proc::env and wfn? @loriab

Reply:
agree, keep `P::e` as is. I'm regarding qcvars on wfn as the opportunity to clean up wrongs we did with `P::e`. eventually direct setting of `P::e` will be deprecated, and wfn vars will be copied up.